integration-tests/test/predeploys.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+
+/* Imports: External */
+import { Contract, Wallet } from 'ethers'
+import { OptimismEnv } from './shared/env'
+import { getContractInterface } from '@eth-optimism/contracts'
+
+describe('ECDSAContractAccount', () => {
+  let l2Wallet: Wallet
+
+  const DEFAULT_TRANSACTION = {
+    to: '0x' + '1234'.repeat(10),
+    gasLimit: 33600000000001,
+    gasPrice: 0,
+    data: '0x',
+    value: 0,
+  }
+  before(async () => {
+    const env = await OptimismEnv.new()
+    l2Wallet = env.l2Wallet
+  })
+
+  let ProxyEOA: Contract
+
+  beforeEach(async () => {
```

Comment:
This could be a `before` instead of a `beforeEach` which would speed up the tests if more `it` blocks are added to this file